### PR TITLE
refactor: refactor contacts sorting for readability

### PR DESCRIPTION
**Pull request by codisart**
First of all thanks so much for taking the time to open a pull request and help the project. It's because of people like you that we love working on this project.

Please read the list below. Feel free to delete this text after but we need you to read it so we make sure that the project is consistent and remains of quality.

#### Before submitting the PR
- [x] Read the [CONTRIBUTING document](https://github.com/monicahq/monica/blob/master/CONTRIBUTING.md) before submitting your PR.
- [x] If the PR is related to an issue or fix one, don't forget to indicate it.
- [x] Create your PR as draft if it is not final yet. Mark it as ready... when it’s ready. Otherwise the PR will be considered complete and rejected if it's not working.

### General checks
- [x] Make sure that the change you propose is the smallest possible.
- [x] The name of the PR should follow the [conventional commits guideline](https://github.com/monicahq/monica/blob/master/docs/contribute/readme.md#conventional-commits) that the project follows.

### CODE REVIEW

**Comment by codisart** [comment]
Could this PR be hacktoberfest accepted ?

File: app/Models/Contact/Contact.php [hunk 1]
```diff
@@ -471,26 +471,30 @@ public function scopeSortedBy(Builder $builder, $criteria)
...
+                return $builder;
         }
     }
 
+    /**
+     * @param Builder $builder
+     * @param string $order
+     * @return Builder
+     */
+    public function sortedByLastActivity(Builder $builder, string $order): Builder
```
Comment (asbiin) [inline app/Models/Contact/Contact.php:487]:
```suggestion
    private function sortedByLastActivity(Builder $builder, string $order): Builder
```

File: app/Models/Contact/Contact.php [hunk 2]
```diff
@@ -471,26 +471,30 @@ public function scopeSortedBy(Builder $builder, $criteria)
...
-                return $builder->orderBy('first_name', 'asc');
+                return $builder;
         }
     }
 
+    /**
+     * @param Builder $builder
+     * @param string $order
+     * @return Builder
+     */
```
Comment (asbiin) [inline app/Models/Contact/Contact.php:486]:
```suggestion
    /**
     * Sort the contacts using last activity.
     * @param Builder $builder
     * @param string $order
     * @return Builder
     */
```

**Review by asbiin — APPROVED** [review]
Great! Just two little changes to do.

**Comment by asbiin** [comment]
Thank you @codisart !

**Pull request merged**